homework-6/script-1.js

```diff
@@ -0,0 +1,78 @@
+//Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+function getMinNumber(num_1, num_2, num_3, num_4) {
+    const result = Math.min(num_1, num_2, num_3, num_4);
+    if (!isNaN(result)) {
+        return result;
+    } else {
+        console.log("I can't figure out min number");
+    }
+}
+getMinNumber("5", 0, 5, 2);
+
+/*Написати функцію, яка приймає 2 аргументи та шукає найбільший спільний дільник для них. 
+Якщо такого числа немає, то повідомляти про відсутність*/
+function getGreatestCommonDivisor(num_1, num_2) {
```

Comment:
Here are some issues with negatives.
You should validate numbers to be greater than zero
<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191334283-6e31b62d-8159-4e8a-bd1a-eb40c11bea3b.png">